packages/manager/modules/emailpro/src/dashboard/emailpro.controller.js

```diff
@@ -512,5 +518,9 @@ export default /* @ngInject */ function EmailProCtrl(
       searchText: $scope.exchange.domain,
     });
 
+  $scope.isMigratedMxPlan = () => {
+    return $scope.exchange?.isMXPlan && $scope?.isMxPlanMigrated;
```

Comment:
```suggestion
    return $scope.exchange?.isMXPlan && $scope.isMxPlanMigrated;
```
It seems unlikely that $scope is null or undefined

Reply:
was a mistake (no need to check  scope... XD)